test/docker-compose/smoke-test.sh

```diff
@@ -23,12 +22,13 @@ fi
 echo "TEST: Checking every 10s that containers are running for 5 minutes..."
 for i in {0..30}; do
 	containers=$(docker ps --format '{{.Names}}' | xargs -I{} -n1 sh -c "printf '{}: ' && docker inspect --format '{{.State.Status}}' {}")
-	containers_running=$(echo "$containers" | grep "running" | wc -l)
+	containers_running=$(echo "$containers" | grep -c "running")
```

Comment:
Wouldnt this make `[ "$containers_running" -ne "$expect_containers" ]` fail? As it expects a count.

PS: I would change all `[]` to `[[]]`

Reply:
```
  -c, --count               print only a count of selected lines per FILE
```